mage_integrations/mage_integrations/sources/paystack/streams/base.py

```diff
@@ -0,0 +1,166 @@
+from datetime import datetime, timedelta
+from typing import Dict, List
+from dateutil.parser import parse
+from mage_integrations.sources.constants import REPLICATION_METHOD_INCREMENTAL
+from mage_integrations.sources.messages import write_schema
+from mage_integrations.sources.paystack.client import PaystackClient
+from mage_integrations.sources.state_utils import (
+    get_last_record_value_for_table,
+    incorporate
+)
+import inspect
+import os
+import pytz
+import singer
+
+
+LOGGER = singer.get_logger()
```

Comment:
Let's pass Paystack's logger to this class. `LOGGER` doesn't log the correct format that can be shown in Mage's logs page.